File: src/my_params/src/param.cpp
#include "ros/ros.h"

int main(int argc, char **argv)
{
	ros::init(argc, argv, "getting_params");

	int int_var;
	double double_var;
	std::string string_var;

	ros::param::get("/my_integer", int_var);
	ros::param::get("/my_float", double_var);
	ros::param::get("/my_string", string_var);

	ROS_INFO("Int: %d, Float: %lf, String: %s", int_var, double_var, string_var.c_str());
}

File: src/my_params/scripts/param.py
#! /usr/bin/env python
import rospy

int_var = rospy.get_param("/my_integer")
float_var = rospy.get_param("/my_float")
string_var = rospy.get_param("/my_string")

print("Int: "+str(int_var)+", Float: "+str(float_var)+" String: "+str(string_var))
